src/Symfony/Component/Serializer/Exception/ExtraAttributesException.php

```diff
@@ -20,9 +20,9 @@ class ExtraAttributesException extends RuntimeException
 {
     private $extraAttributes;
 
-    public function __construct(array $extraAttributes, \Throwable $previous = null)
+    public function __construct(string $class, array $extraAttributes, \Throwable $previous = null)
```

Comment:
this is a BC break, we won't allow it on any branch
the class could be added last I suppose? that would fix the BC break.

Reply:
Yes, as discussed above. Just makes the constructor a bit ugly but 🤷‍♂️

Reply:
The ugliest thing to do would be to break BC :)

Reply:
On way to work around both the BC break and the ugliness would be to create a setter instead (`setClass()`).

Reply:
I still think that having a `setClass()` method is the best option to move forward.